tests/test_main.py

```diff
@@ -726,14 +726,13 @@ def test_over_score_thresh(m):
     """A user might want to change the config after model training and update the score thresh"""
     img = get_data("OSBS_029.png")
     original_score_thresh = m.model.score_thresh
-    high_thresh = 0.6
-    m.model.score_thresh = high_thresh
+    m.model.score_thresh = 0.8
 
     # trigger update
     boxes = m.predict_image(path=img)
 
-    assert all(boxes.score > high_thresh)
-    assert m.model.score_thresh == high_thresh
+    assert all(boxes.score > 0.8)
+    assert m.model.score_thresh == 0.8
```

Comment:
Why are these changes being made? This seems like a really odd response to any of my comments and should be reverted.

Reply:
i reverted this.